### ISSUE TITLE
Add 'accessibility_caption' field to medias

### ISSUE THREAD

**Issue by zetalp** [pgrimaud/instagram-user-feed#83]
"node" has a property "accessibility_caption"

**Comment by pgrimaud** [comment]
Sure, I will add this properties tomorrow.

**Comment by pgrimaud** [comment]
Hello @zetalp 

I added accessibility_caption in version 6.2.4

**Comment by zetalp** [comment]
Thank you for adding the field!

There are 2 small issues though:

1.  'accessibilityCaption' is missing from the toArray() function
2.  'accessibilityCaption' is always null for all entries that are not the first 12 medias

**Comment by pgrimaud** [comment]
@zetalp I fix missing data from toArray() method here : https://github.com/pgrimaud/instagram-user-feed/commit/c3498f588782fe41c00bd5798df2027c3e83c32a

But I can't find any account with accessibility_caption filled for medias after the 12th. Could you give me an example?
Thanks

**Comment by zetalp** [comment]
>But I can't find any account with accessibility_caption filled for medias after the 12th. Could you give me an example?

I can't find either. I am a novice and thought it had to do with the package somehow not retaining the 'accessibility_caption' value after the first 12 medias.

Thank you!

**Comment by pgrimaud** [comment]
I guess it's an Instagram limitation
